docs/applications/containers/kubernetes-reference/index.md

```diff
@@ -17,7 +17,7 @@ external_resources:
 - '[Kubernetes Concepts Documentation](https://kubernetes.io/docs/concepts/)'
 ---
 
-This is a reference for common, frequently used Kubernetes terminology.
+This is a reference for common Kubernetes terminology.
```

Comment:
Felt like common and frequently used were close enough in meaning to drop one of them